crates/goose/src/config/extensions.rs

```diff
@@ -29,10 +30,29 @@ pub struct ExtensionConfigManager;
 
 impl ExtensionConfigManager {
     fn get_extensions_map() -> Result<HashMap<String, ExtensionEntry>> {
-        let config = Config::global();
-        Ok(config
+        let mut extensions_map = Config::global()
             .get_param(EXTENSIONS_CONFIG_KEY)
-            .unwrap_or_else(|_| HashMap::new()))
+            .unwrap_or_else(|_| HashMap::new());
+
+        if !extensions_map.is_empty() {
```

Comment:
Why do we only enable the platform extensions if the user has existing extensions?

Reply:
good question; I saw this happening where it would get here and not have any extensions yet and then if we insert it here, something else is not going to set their defaults. obviously would be better to solve the issue in that something else, but that feels like it is inside some hairy client code. we should get a good opinion though on where in general the default extensions are set. I'm not sure !